save-core/src/commonNonJsTest/kotlin/org/cqfn/save/core/integration/WarnDirTest.kt

```diff
@@ -15,7 +15,7 @@ class WarnDirTest {
 
     @Test
     fun `execute warn plugin on the directory chapter2`() =
-            runTestsWithDiktat(listOf("warn-dir/chapter2"), 1)
+            runTestsWithDiktat(listOf("warn-dir/chapter2/GenericFunctionTest.kt"), 1)
```

Comment:
Why did you change this?

I suggest to create separate test, if you would like to check the logic with individual test file

Reply:
Forgot to delete this